sway-fmt-v2/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
```

Comment:
If we're publishing this to crates.io, can you add it to https://github.com/FuelLabs/sway/blob/c253486ed03414e851a6a43c5084b52d337979a4/.github/workflows/ci.yml#L468-L479

Reply:
Perhaps we should not publish `sway-fmt-v2` and instead wait until it's at feature parity with the existing `sway-fmt`, then replace the old `sway-fmt`, then publish this as `sway-fmt` instead?

In the meantime, we can always `cargo install --path ./sway-fmt-v2` if we want to use/test it locally or in CI.

Maybe we should add a `publish = false` for now instead? Thoughts @eureka-cpu @adlerjohn ?

Reply:
Not against that. If we're not publishing then set the version to `0.0.0` as well